packages/react/src/Breadcrumbs/Breadcrumbs.module.css

```
@@ -9,6 +10,100 @@
   margin-bottom: 0;
 }
 
+[data-overflow='menu'] .BreadcrumbsList,
+[data-overflow='menu-with-root'] .BreadcrumbsList {
+  white-space: nowrap;
+  display: flex;
+  flex-direction: row;
+}
+
+.BreadcrumbsItem {
+  display: inline-grid;
+  grid-auto-flow: column;
+  align-items: center;
+  flex: 0 99999 auto;
+  min-width: auto;
+  font-size: var(--text-body-size-medium);
+  white-space: nowrap;
+  list-style: none;
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &:last-child {
+    .ItemSeparator {
+      display: none;
+    }
+  }
+}
+
+[data-overflow='menu'] .Item,
+[data-overflow='menu-with-root'] .Item {
+  display: inline-block;
+  font-size: var(--text-body-size-medium);
+  color: var(--fgColor-link);
+  text-decoration: none;
+  padding-inline: var(--base-size-6);
+  padding-block: var(--base-size-4);
+  border-radius: var(--borderRadius-medium);
+
+  &:hover {
+    background: var(--control-transparent-bgColor-hover);
+  }
+
+  &:focus {
+    box-shadow: none;
+    outline: 2px solid var(--focus-outlineColor, var(--color-accent-fg));
+    outline-offset: -2px;
+  }
+}
+
+.MenuSummary {
+  list-style: none;
+  display: inline-block;
+  outline: none;
+  height: 0.5rem;
+}
+
+.MenuSummary::-webkit-details-marker {
+  display: none;
+}
+
+.MenuDetails {
+  position: relative;
+  display: inline-block;
+}
+
+.MenuDetails summary {
+  list-style: none;
+  cursor: pointer;
+  outline: none;
+}
+
+.MenuDetails summary::-webkit-details-marker {
+  display: none;
+}
+
+.MenuOverlay {
+  position: absolute;
+  z-index: 1;
+  box-shadow: var(--shadow-resting-medium);
+  border-radius: var(--borderRadius-large);
+  background-color: var(--overlay-bgColor);
+  list-style: none;
+  width: max-content;
```

```suggestion
  min-width: var(--overlay-width-xsmall);
  max-height: 100vh;
  max-width: var(--overlay-width-small);
  overflow: hidden;
```

Just aligning with Overlay